Arrangement_on_surface_2/doc/Arrangement_on_surface_2/Concepts/AosTraits--ApproximateUnbounded_2.h

```diff
@@ -0,0 +1,49 @@
+namespace AosTraits {
+
+/*! \ingroup PkgArrangementOnSurface2ConceptsFunctionObjects
+ * \cgalConcept
+ *
+ * \cgalRefines{Approximate_2}
+ *
+ * \cgalHasModelsBegin
+ * \cgalHasModels{AosApproximatePointTraits_2::Approximate_2}
+ * \cgalHasModels{AosApproximateTraits_2::Approximate_2}
+ * \cgalHasModels{AosApproximateUnboundedTraits_2::Approximate_2}
+ * \cgalHasModelsEnd
+ */
+class ApproximateUnbounded_2 {
+public:
+  /// \name Operations
+  /// A model of this concept must provide:
+  /// @{
+
+  /*! approximates a given \f$x\f$-monotone curve constrained to a bounding
+   * box. It computes one or more sequences of approximate points that represent
+   * the disconnected portions of a polyline that approximates `xcv` within the
+   * bounding box `bbox`, and inserts them into output containers given through
+   * the output iterator `oi`.  The first point of the first sequence and the
+   * last point of the last sequence are always approximations of the endpoints
+   * of the given curve.
+   *
+   * \param xcv The exact \f$x\f$-monotone curve.
+   * \param error The error bound of the polyline approximation. This is the
+   *        Hausdorff distance between the curve and the polyline that
+   *        approximates the curve.
+   * \param oi An output iterator for the output containers.
+   * \param bbox the bounding box.
+   * \param l2r A Boolean flag that indicates whether the curve direction is
```

Comment:
```suggestion
   * \param l2r A %Boolean flag that indicates whether the curve direction is
```

Reply:
Why is it needed? "Boolean" does not create an undesired link.
I've searched for "Boolean" in txt files and found 233 occurrences out of which only 2 have the '%' as a prefix and only in the Polygon_repair/doc/Polygon_repair/Polygon_repair.txt file.
I would guess that the `%` character should be removed. Am I missing something?

Reply:
I had the wrong idea that we have a type `CGAL::Boolean`. Sorry for the noise.

Reply:
And yes, it can be removed.